README.md

```diff
@@ -14,39 +14,46 @@ rosdep update
 
 # Clone repo
 cd <catkin workspace location>/src
-git clone --recurse-submodules https://github.com/uwrobotics/uwrt_mars_rover.git
+git clone https://github.com/uwrobotics/uwrt_mars_rover.git
 
-# Install all dependencies
-rosinstall --catkin . uwrt_mars_rover/uwrt.rosinstall
+# Install upstream source dependencies
+rosinstall --catkin . uwrt_mars_rover/upstream_dependencies.rosinstall
+
+# Install metapackage source dependencies
+rosinstall --catkin . uwrt_mars_rover/metapackage_dependencies.rosinstall
+
+# Install all binary dependencies
 rosdep install --from-paths . --ignore-src -r -y
 ```
 
-## Dependencies
-For ease of setup on the Jetson, we should always try to use prebuilt binaries of dependent ROS packages, instead of 
-including their sources. This means that if a package you write is dependent on a 3rd party package, you should just 
-declare that dependency in the `CMakelists.txt` and `package.xml`. `rosdep` will take care of resolving the 
-dependencies. If you need to update binary dependencies, run the following commands:
+If you're setting up the repository on the Nvidia Jetson, you also need to install the arm64 upstream dependencies:
+```bash
+# Install upstream source dependencies for arm64
+rosinstall --catkin . uwrt_mars_rover/arm64_upstream_dependencies.rosinstall
+```
+
+## Updating Dependencies
+Commands to update dependencies:
 ```bash
 cd <catkin workspace location>/src
+rosinstall --catkin . 
 rosdep install --from-paths . --ignore-src -r -y
 ```
 
+
+## Adding Dependencies
+For ease of setup on the Jetson, we should always try to use prebuilt binaries of dependent ROS packages, instead of 
+including their sources. This means that if a package you write is dependent on a 3rd party package, you should just 
+declare that dependency in the `CMakelists.txt` and `package.xml`. `rosdep` will take care of resolving the 
+dependencies. 
+
 In the event that you cannot use the binaries (ex. we rely on a feature that has not been released), the package source 
 code should be cloned outside of our metapackage, so that our CI doesn't run linting/formatting checks on it. To do 
 this, declare the source dependency in `uwrt.rosinstall`. If using unreleased features(ie. cloning 3rd party master branch), 
```

Comment:
Should the `uwrt.rosinstall` be replaced with `upstream_dependencies.rosinstall` here?

Reply:
yes. this is a typo